docs/contributing/marketplace.md

```diff
@@ -64,10 +64,15 @@ If you have already submitted your contribution and you would like to make chang
 
 1. Add or remove content items from the pack, as needed.
 
-1. Press **Save and Contribute** and complete the form as documented [above](#submit-a-content-pack).
- 
    **Note**: Changing the pack name or the email of the contributor at this stage will result in creating a pull request on GitHub, rather than updating the existing one.
    
+   **When resubmitting to an existing pack, make sure the contribution name is the same as the display name of the pack you are contributing to.**
```

Comment:
What is "contribution name"?

Reply:
TBH I don't think I understand this myself.

Reply:
By saying "Contribution name" I've tried to describe the title/name that was given to the contribution by the user - In the xsoar UI you can find the names here: 
![contribution_name2](https://user-images.githubusercontent.com/82749224/197382341-eb015f39-7b29-4ed9-b9ee-838024e170e8.png)